src/components/sideBarContentEdit/SelectField.tsx

```diff
@@ -15,6 +16,17 @@ type Props = {
   userCanAddToList?: boolean;
 };
 
+type PropsDirectory = {
+  name: string;
+  editedData: string;
+  index: number;
+  path: string;
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Why do we have this eslint? Is this prop not being used?

Reply:
> Why do we have this eslint? Is this prop not being used?

It is being used but on line 25 the parameter is not being return and esLint throws a warning for that